build.gradle.kts

```diff
@@ -84,7 +84,7 @@ allprojects {
             pom {
                 description.set("Simple, reliable, interoperable. A better RPC.")
                 name.set("connect-library") // This is overwritten in subprojects.
-                group = "build.buf"
+                group = "com.connectrpc"
                 url.set("https://github.com/bufbuild/connect-kotlin")
```

Comment:
Are we going to change this URL once we've relocated the repo to the new home?

Reply:
ya i think we do that change once it's relocated to that repo? what do you think?

Reply:
It all depends on when we merge it. I think we should do one final release at this location before copying to a new repo in connectrpc (so that the only changes in the version after we relocated are related to the relocation itself). We'll also need to build one more release at the old location for the relocation to take effect.